pkg/phlaredb/block/block_test.go

```diff
@@ -84,7 +84,7 @@ func TestDelete(t *testing.T) {
 
 	bkt := objstore.NewInMemBucket()
 	{
-		meta, dir, err := block_testutil.CreateBlock(t, func() []*testhelper.ProfileBuilder {
```

Comment:
I think block delete is not deleting the symdb prefix within the block. Can you double check:

```
gsutil ls gs://dev-us-central-0-profiles-dev-001-data/1218/phlaredb/01H9YTWD4JS697B9F2RRYGB9VA/symbols
gs://dev-us-central-0-profiles-dev-001-data/1218/phlaredb/01H9YTWD4JS697B9F2RRYGB9VA/symbols/index.symdb
gs://dev-us-central-0-profiles-dev-001-data/1218/phlaredb/01H9YTWD4JS697B9F2RRYGB9VA/symbols/stacktraces.symdb
```

Reply:
yeah I can see the Delete being not recursive. Let me work on that. Nice catch !

Reply:
Can't find any issue I wonder if this is because that cluster is in a crashloop